### DISCUSSION TITLE
Welcome to InsGallery Discussions!

### DISCUSSION THREAD

**Discussion by JessYanCoding** [JessYanCoding/InsGallery#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JessYanCoding** [comment]
...

**Reply by nede64** [reply]
cool stuff, will this ever get an update ?

**Comment by nede64** [comment]
cool stuff, will this ever get an update ?
